tutorials/mpi-hello-world/index.md

```diff
@@ -3,6 +3,7 @@ layout: post
 title: MPI Hello World
 author: Wes Kendall
 categories: Beginner MPI
+translations: zh_cn
```

Comment:
You'll need to add the `folder: ` header as well or I don't think the links will turn out correctly.

Reply:
You are right. But rather than adding a folder string, I made some changes to the javascript part so that it looks for the site url directly.

Reply:
Oh ok. I see that change now. I'll make sure all of that is good then.